src/index.js

```diff
@@ -22,6 +22,7 @@ export { default as Section } from './ui/Section/Section';
 export { default as SocialIcon } from './ui/SocialIcon/SocialIcon';
 export { default as Text } from './ui/Text/Text';
 export { default as Filters } from './ui/Filters/Filters';
+export { default as Player } from './ui/Video/Player';
```

Comment:
The export should be the same as the name of the component: `export { default as VideoPlayer } from  './ui/VideoPlayer/VideoPlayer';`